src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
@debloper , 
that looks good solution , but according to @nimishamukherjee , core is fixing this thing from backend , in this case if there is fullName = null they will return github username.
I am still waiting for @aslakknutsen for the backend implementation.
If am not wrong , you are suggesting workItem.relationalData.creator.attributes.userName to be replaced by "No creator found" ?